core/prover/src/main.rs

```diff
@@ -22,37 +25,98 @@ fn main() {
     signal_hook::flag::register(signal_hook::SIGQUIT, Arc::clone(&stop_signal))
         .expect("Error setting SIGQUIT handler");
 
-    let worker = env::var("POD_NAME").unwrap_or_else(|_| "default".to_string());
-    let key_dir = std::env::var("KEY_DIR").expect("KEY_DIR not set");
-    info!("creating prover, worker: {}", worker);
-    let mut prover = BabyProver::create(worker, key_dir).unwrap();
-    let prover_id = prover.prover_id;
+    let worker_name = env::var("POD_NAME").unwrap_or_else(|_| "default".to_string());
```

Comment:
what if I would like to run 2 provers on my machine to test it?

Reply:
I guess you'd have to have two different envs with different `POD_NAME`. What would you like?

Reply:
if there is no pod name -> hostname + random id or something like that

Reply:
Lets make it required for now.